src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,35 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (obj instanceof Rectangle) {
```

Comment:
what is the difference between `instanceof` and `getClass()`?

Reply:
How about this comment?
![image](https://user-images.githubusercontent.com/36365171/70231970-0ff36800-1764-11ea-977e-d5420cc53c4d.png)
https://javarush.ru/groups/posts/1340-peregruzka-metodov-equals-i-hashcode-v-java